kubernetes/README.md

```diff
@@ -0,0 +1,815 @@
+
+    * Look up the screenshots above for the expected AWS Console UI for Security group & EFS. If you dont see the Ingress permissions / Mount points created, Execute steps from ```setup_efs.sh``` to make sure that they complete as expected.  We need 1 Mount point for every region where Nodes would be deployed. *This step is very critical to the setup* . If you run to any errors the `aws-efs-csi` driver might throw errors which might be hard to diagonize.
+
+  * EFS CSI Driver installation
+
+    * Ensure that ```--set efsProvisioner.efsFileSystemId=YOUR-EFS-FS-ID --set efsProvisioner.awsRegion=us-west-2 --set efsProvisioner.reclaimPolicy=Retain --generate-name``` is set correctly 
+
+    * You may inspect the values by running ``helm list`` and ```helm get all YOUR_RELEASE_ID``` to verify if the values used for the installation
+
+    * You can execute the following commands to inspect the pods / events to debug EFS / CSI Issues
+
+      ```bash
+      kubectl get events --sort-by='.metadata.creationTimestamp'
+      
+      kubectl get pod --all-namespaces # Get the Pod ID
+      
+      kubectl logs pod/efs-provisioner-YOUR_POD
+      kubectl logs pod/efs-provisioner-YOUR_POD
+      kubectl describe pod/efs-provisioner-YOUR_POD
+      ```
+
+    * A more involved debugging step would involve installing a simple example app to verify EFS / EKS setup as described [here](https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html) (Section : *To deploy a sample application and verify that the CSI driver is working*)
+
+    * More info about the driver can be found at 
+
+      * [Github Page](https://github.com/kubernetes-sigs/aws-efs-csi-driver/) / [Helm Chart](https://github.com/kubernetes-incubator/external-storage/tree/master/aws/efs) / [EKS Workshop](https://www.eksworkshop.com/beginner/190_efs/efs-provisioner/) / [AWS Docs](https://aws.amazon.com/premiumsupport/knowledge-center/eks-persistent-storage/)
+
+  
+
+  
+
+  **Troubleshooting Torchserve Helm Chart**
+
+  
+
+  Possible errors in this step may be a result of 
+
+  * Incorrect values in ``values.yaml``
+    * Changing values in `torchserve.pvd_mount`  would need corresponding change in `config.properties`
+  * Invalid `config.properties`
+    * You can verify these values by running this for local TS installation
+  * TS Pods in *Pending* state
+    * Ensure you have available Nodes in Node Group
+
+  * Helm Installation
+    * You may inspect the values by running ``helm list`` and `helm get all ts` to verify if the values used for the installation
+    * You can uninstall / reinstall the helm chart by executing  `helm uninstall ts` and `helm install ts .`
+    * If you get an error `invalid: data: Too long: must have at most 1048576 characters`, ensure that you dont have any stale files in your kubernetes dir. Else add them to .helmignore file.
+
+  
+
+  ## Deleting Resources
+
+  
+
+  * Delete EKS cluster `eksctl delete cluster -name YOUR_CLUSTER_NAME`
```

Comment:
@dhanainme It should with `--name` with two hyphens

Reply:
Done